java/src/processing/mode/java/JavaBuild.java

```diff
@@ -1083,7 +1083,9 @@ static public String[] getArgsJavaFX(String modulePath) {
       // Full list of modules, let's not commit to all of these unless
       // a compelling argument is made or a reason presents itself.
       //"javafx.base,javafx.controls,javafx.fxml,javafx.graphics,javafx.media,javafx.swing,javafx.web"
-      "--add-modules", "javafx.base,javafx.graphics,javafx.swing",
+     // "--add-modules", "javafx.base,javafx.graphics,javafx.swing",
+// ***** change to:
```

Comment:
please remove the `change to` line and remove the old line, don't comment it out